pkg/types/baremetal/validation/platform.go

```diff
@@ -243,6 +243,23 @@ func validateHostsBMCOnly(hosts []*baremetal.Host, fldPath *field.Path) field.Er
 	})
 }
 
+// validateHostsBootMACAddress validates all hosts BootMACAddress are valid or not
+func validateHostsBootMACAddress(hosts []*baremetal.Host, fldPath *field.Path) field.ErrorList {
+	hostErrs := field.ErrorList{}
+
+	for idx, host := range hosts {
+		if host.BootMACAddress == "" {
```

Comment:
There is already a couple of validation rule on BootMacAddress field, in particular the `required` one should ensure that  the field is not empty: https://github.com/openshift/installer/blob/969378dc498336d1ac29baa89f75324a44467d5d/pkg/types/baremetal/platform.go#L39

Reply:
Thanks for review @andfasano.

That is interesting, because it is optional in upstream https://github.com/metal3-io/baremetal-operator/blob/3612e86973f1ce0fef606557c7e20dd482d8fddc/config/crd/bases/metal3.io_baremetalhosts.yaml#L108.

I guess, this is intentionally marked as required in OpenShift?

Reply:
Yes, originally it was set in https://github.com/openshift/installer/pull/3208